CMakeLists.txt

```diff
@@ -4,7 +4,18 @@ cmake_minimum_required(VERSION 3.4)
 
 include(ExternalProject)
 
-project( CCTag )
+project(CCTag VERSION 1.0.0 LANGUAGES C CXX)
```

Comment:
Yes, the code is no more autonomous, just for that. So your IDE or static analysis tools will make an error on `#include "cctag/version.hpp"` as this file doesn't exist in your source tree.
I prefer to keep the important information in the code instead of cmake. But it's a detail, both works.